### DISCUSSION TITLE
Official provider terraform timelines...

### DISCUSSION THREAD

**Discussion by tech5678910** [iwarapter/terraform-provider-pingaccess#127] [Q&A]
Planning to suggest use this provider and terraform for our ping project in a large organization...

I am quite new to the ping products :-) , have two questions...

1. How much of the ping access capabilities does the provider/resources/data sources cover ?

2. Also any news on becoming the official provider for ping access in terraform sooner ?

**Comment by iwarapter** [comment]
Hi @tech5678910

1. The [registry contains the documentation](https://registry.terraform.io/providers/iwarapter/pingaccess/latest/docs) for all supported resources/data sources -  if there is anything missing feel free to raise a [feature request](https://github.com/iwarapter/terraform-provider-pingaccess/issues/new?assignees=&labels=enhancement&template=---feature-request.md&title=) 
2. I know of a few discussions PingIdentity but I don't know how far they went.
